### PR TITLE
Add Arcata, CA

### PR DESCRIPTION
**Pull request by tunglinn**
First community map!

### CODE REVIEW

**Comment by Eric-Arellano** [comment]
Yay! Your change has a lot of unrelated formatting changes. Try running `npm install` then `npm run fmt`.

File: data/city-stats.json [hunk 1]
```diff
@@ -49,6 +49,17 @@
...
   },
+  "arcata-ca": {
+    "name": "Arcata, CA",
+    "percentage": "34%",
+    "cityType": "suburb",
+    "population": "18,857",
+    "urbanizedAreaPopulation": "19,714",
+    "parkingScore": null,
+    "reforms": "no reforms",
+    "url": null,
```
Comment (Eric-Arellano) [inline data/city-stats.json:60]:
```suggestion
    "url": "https://parkingreform.org/mandates-map/city_detail/Arcata_CA.html",
```

File: data/city-stats.json [hunk 2]
```diff
@@ -49,6 +49,16 @@
...
     "url": "https://parkingreform.org/mandates-map/city_detail/Anchorage_AK.html"
   },
+  "arcata-ca": {
+    "name": "Arcata, CA",
+    "percentage": "26%",
+    "cityType": "suburb",
+    "population": "18,857",
+    "urbanizedAreaPopulation": "19,714",
+    "parkingScore": null,
+    "reforms": "no reforms",
```
Comment (Eric-Arellano) [inline data/city-stats.json:59]:
```suggestion
    "reforms": "implemented",
```

File: data/city-stats.json [hunk 3]
```diff
@@ -49,6 +49,16 @@
     "reforms": "implemented",
     "url": "https://parkingreform.org/mandates-map/city_detail/Anchorage_AK.html"
   },
+  "arcata-ca": {
+    "name": "Arcata, CA",
+    "percentage": "26%",
+    "cityType": "suburb",
+    "population": "18,857",
+    "urbanizedAreaPopulation": "19,714",
```
Comment (Eric-Arellano) [inline data/city-stats.json:57]: How did you get this number?

**Review by Eric-Arellano — APPROVED** [review]
Exciting!

File: data/city-stats.json (hunk 3 above)
Comment (tunglinn) [reply data/city-stats.json:57]: Submitted along with the maps. Will double check with Tony if we need to double check before.

**Pull request merged**